platform_events-handler/components/config.js

```diff
@@ -0,0 +1,19 @@
+var getStageConfig = (context) => {
```

Comment:
We have a newer version of this file (you have referenced it down below). To be consistent can you replace this with that?